cdpacctest/acctest.go

```diff
@@ -95,11 +105,25 @@ provider "cdp" {
 `
 }
 
-func TestAccAwsProviderConfig() string {
-	return `
-provider "aws" {
+type awsProvider struct {
+	profile string
+	region  string
 }
-`
+
+func NewAwsProvider(profile, region string) awsProvider {
```

Comment:
there is a bit of a contradiction here too: `NewAwsProvider` is exported but the type.

Reply:
we can fix it later

Reply:
OK lets discuss this tomorrow